decode.go

```diff
@@ -706,47 +730,49 @@ type DecMode interface {
 }
 
 type decMode struct {
-	tags                  tagProvider
-	dupMapKey             DupMapKeyMode
-	timeTag               DecTagMode
-	maxNestedLevels       int
-	maxArrayElements      int
-	maxMapPairs           int
-	indefLength           IndefLengthMode
-	tagsMd                TagsMode
-	intDec                IntDecMode
-	mapKeyByteString      MapKeyByteStringMode
-	extraReturnErrors     ExtraDecErrorCond
-	defaultMapType        reflect.Type
-	utf8                  UTF8Mode
-	fieldNameMatching     FieldNameMatchingMode
-	bigIntDec             BigIntDecMode
-	defaultByteStringType reflect.Type
-	byteStringToString    ByteStringToStringMode
-	fieldNameByteString   FieldNameByteStringMode
+	tags                    tagProvider
+	dupMapKey               DupMapKeyMode
+	timeTag                 DecTagMode
+	maxNestedLevels         int
+	maxArrayElements        int
+	maxMapPairs             int
+	indefLength             IndefLengthMode
+	tagsMd                  TagsMode
+	intDec                  IntDecMode
+	mapKeyByteString        MapKeyByteStringMode
+	extraReturnErrors       ExtraDecErrorCond
+	defaultMapType          reflect.Type
+	utf8                    UTF8Mode
+	fieldNameMatching       FieldNameMatchingMode
+	bigIntDec               BigIntDecMode
+	defaultByteStringType   reflect.Type
+	byteStringToString      ByteStringToStringMode
+	fieldNameByteString     FieldNameByteStringMode
+	returnTypeForEmptyIface ReturnTypeForEmptyInterface
```

Comment:
Maybe rename this.

```suggestion
	UnrecognizedTagToAny UnrecognizedTagToAnyMode
```

Reply:
Thanks for the suggestion. I've made these changes.